src/strategy/BaseStrategy.sol

```diff
@@ -104,7 +104,7 @@ abstract contract BaseStrategy is BaseVault, IBaseStrategy {
      * @return maxAssets The maximum amount of assets.
      */
     function _maxWithdrawAsset(address asset_, address owner) internal view virtual returns (uint256 maxAssets) {
-        if (paused() || !_getAssetStorage().assets[asset_].active) {
+        if (paused() || !_getBaseStrategyStorage().isAssetWithdrawable[asset_]) {
```

Comment:
let's move this fix in a separate clean Pull request please with tests that are unit and integration that verify cover this

Reply:
can we please revert this change here